mzLib/FlashLFQ/IndexedMassSpectralPeak.cs

```diff
@@ -29,7 +29,7 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            return Mz.GetHashCode();
+            return HashCode.Combine(Mz.GetHashCode(), ZeroBasedMs1ScanIndex.GetHashCode());
```

Comment:
Minor point, mostly of curiosity: do you need to hash an index or is the index unique enough?

Reply:
The index probably doesn't need to be converted, you're right. I actually think you can pass in a double (mz) and an int (index) and get a hashcode out, elminating 2/3 hash operations